eclair-core/src/main/resources/reference.conf

```diff
@@ -103,6 +103,12 @@ eclair {
       max-cltv =  1008             // max acceptable cltv expiry for the payment (1008 ~ 1 week)
       fee-threshold-sat = 21       // if fee is below this value we skip the max-fee-pct check
       max-fee-pct = 0.03           // route will be discarded if fee is above this value (in percentage relative to the total payment amount); doesn't apply if fee < fee-threshold-sat
+
+      // channel 'weight' is computed with the following formula: channelFee * (cltvDelta * ratio-cltv + channelAge * ratio-channel-age + channelCapacity * ratio-channel-capacity)
+      // the following parameters can be used to ask the router to use heuristics to find i.e: 'cltv-optimized' routes
+      ratio-cltv = 0.0             // when computing the weight for a channel, consider its CLTV delta in this proportion
```

Comment:
We should document valid bounds for these ratio (between 0 and 1 and sum is 1 for example)

Reply:
Done in 0acbc5d